ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -82,6 +90,15 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
       environment.trainingUrl + '/static/data-researcher.html?saml=on');
   }
 
+  async componentDidMount() {
+    const {profileState: {profile: {verifiedInstitutionalAffiliation}}} = this.props;
+    if (verifiedInstitutionalAffiliation) {
+      await institutionApi().getInstitution(verifiedInstitutionalAffiliation.institutionShortName).then(institution =>
```

Comment:
File this under "unrelated nitpicks you reserved the right to ignore", but IMO this might be cleaner if you do the data join on the backend and added a new optional read-only field in the Profile API response.

Reply:
If we include the Institute inline in the profile it would look like this, which feels unwieldy to me:
```
{ 
...
  verifiedInstitutionalAffiliation: {
    shortName: "Broad",
    displayName: "The Broad Institute",
    emailDomains: [ "broad.org", "broad.edu", ... ],
    emailAddress: ["alice@broad-collabs.net", "bob@sanger.uk", "clyde@vumc.org", ..]
  }
}
```

vs:
```
{
...
  verifiedInstitutionalAffiliation: "Broad"
}
```

Reply:
I agree, that's probably too much. But if all we're using in the UI is the display name, you might just pluck that field out and copy it into the Profile response (and it may be useful elsewhere in our UI as well).

Reply:
Done.  Easy enough with MapStruct: https://github.com/all-of-us/workbench/pull/3181/commits/3f629454e746693c97f8de17b5c29b8a2d9edc3d